pkg/cosi/cosi_bucket_access.go

```diff
@@ -0,0 +1,316 @@
+
+	cosiBucketAccessClaim := util.KubeObject(bundle.File_deploy_cosi_bucket_access_claim_yaml).(*nbv1.COSIBucketAccessClaim)
+	cosiBucketAccessClaim.Name = args[0]
+	cosiBucketAccessClaim.Namespace = options.Namespace
+	if appNamespace != "" {
+		cosiBucketAccessClaim.Namespace = appNamespace
+	}
+
+	if !util.KubeDelete(cosiBucketAccessClaim) {
+		log.Fatalf(`❌ Could not delete COSI bucket claim %q in namespace %q`,
+			cosiBucketAccessClaim.Name, cosiBucketAccessClaim.Namespace)
+	}
+}
+
+// RunStatusBucketAccessClaim runs a CLI command
+func RunStatusBucketAccessClaim(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`Missing expected arguments: <bucket-access-claim-name> %s`, cmd.UsageString())
+	}
+
+	appNamespace, _ := cmd.Flags().GetString("app-namespace")
+
+	cosiBucketAccessClaim := util.KubeObject(bundle.File_deploy_cosi_bucket_access_claim_yaml).(*nbv1.COSIBucketAccessClaim)
+	secret := util.KubeObject(bundle.File_deploy_internal_secret_empty_yaml).(*corev1.Secret)
+
+	cosiBucketAccessClaim.Name = args[0]
+	cosiBucketAccessClaim.Namespace = options.Namespace
+	secret.Namespace = options.Namespace
+	if appNamespace != "" {
+		cosiBucketAccessClaim.Namespace = appNamespace
+		secret.Namespace = appNamespace
+
+	}
+
+	if !util.KubeCheck(cosiBucketAccessClaim) {
+		log.Fatalf(`❌ Could not find COSI bucket access claim %q in namespace %q`, cosiBucketAccessClaim.Name, cosiBucketAccessClaim.Namespace)
+	}
+
+	bucketAccessClass := &nbv1.COSIBucketAccessClass{
+		TypeMeta: metav1.TypeMeta{Kind: "BucketAccessClass"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: cosiBucketAccessClaim.Spec.BucketAccessClassName,
+		},
+	}
+
+	if bucketAccessClass.Name != "" {
+		if !util.KubeCheck(bucketAccessClass) {
+			log.Errorf(`❌ Could not get BucketAccessClass %s`, bucketAccessClass.Name)
+		}
+	}
+	secret.Name = cosiBucketAccessClaim.Spec.CredentialsSecretName
+	if !util.KubeCheck(secret) {
+		log.Fatalf(`❌ Could not find COSI bucket access claim secret %q in namespace %q`, secret.Name, cosiBucketAccessClaim.Namespace)
+	}
+
+	sysClient, err := system.Connect(true)
+	if err != nil {
+		util.Logger().Fatalf("❌ %s", err)
+	}
+	var a *nb.AccountInfo
+	if cosiBucketAccessClaim.Status.AccountID != "" {
+		nbClient := sysClient.NBClient
+		account, err := nbClient.ReadAccountAPI(nb.ReadAccountParams{Email: cosiBucketAccessClaim.Status.AccountID})
+		if err == nil {
+			a = &account
+		}
+	}
+
+	fmt.Printf("\n")
+	fmt.Printf("COSI BucketAccessClaim info:\n")
+	fmt.Printf("  %-22s : %t\n", "Bucket Access Granted", cosiBucketAccessClaim.Status.AccessGranted)
+	fmt.Printf("  %-22s : kubectl get -n %s bucketaccessclaim %s\n", "COSIBucketAccessClaim", cosiBucketAccessClaim.Namespace, cosiBucketAccessClaim.Name)
+	fmt.Printf("  %-22s : kubectl get bucketclasses.objectstorage.k8s.io %s\n", "BucketAccessClass", bucketAccessClass.Name)
+	fmt.Printf("\n")
+
+	fmt.Printf("Shell commands:\n")
```

Comment:
you probably want to add this: 
fmt.Printf("  %-22s : alias s3='%saws s3 --no-verify-ssl --endpoint-url %s'\n", "AWS S3 Alias", credsEnv, sysClient.S3URL.String())
and to move credsEnv up so it can use it